web/src/main/java/org/akaza/openclinica/web/filter/OpenClinicaUsernamePasswordAuthenticationFilter.java

```diff
@@ -108,6 +110,28 @@ public Authentication attemptAuthentication(HttpServletRequest request, HttpServ
 
         // Place the last username attempted into HttpSession for views
         HttpSession session = request.getSession(false);
+        
+        
+        /**
+         *  OC-12286
+         */
+        // check the user in the current session
+        
+        UserAccountBean currentSessionUser = null;
+        currentSessionUser = (UserAccountBean) session.getAttribute(SecureController.USER_BEAN_NAME);
+        
+        if(currentSessionUser !=null && !(currentSessionUser.getName().isEmpty()) && !(currentSessionUser.getName().equals(username))) {
+        	//setFilterProcessesUrl("/pages/login/login?action=errorSessionLocked");
+        	try {
+				response.sendRedirect(request.getContextPath() +"/pages/login/login?action=errorSessionLocked");
+			} catch (IOException e) {
+				// TODO Auto-generated catch block
```

Comment:
instead of printing stack trace like this, use a logger.

Reply:
done